Britannia/localization/english/event_localization/BTWK_culture_events_l_english.yml

```diff
@@ -1,4 +1,20 @@
 ﻿l_english:
- culture_event.002.t:0 "" #Can't think of anything
+ culture_event.002.t:0 "The Frisian language"
  culture_event.002.desc:0 "Language is always changing - and so is the Frisian one. After hundreds of years of changes, the new names of our people and our titles have become unrecognizable compared to earlier forms."
```

Comment:
Maybe "have become unrecognizable compared to earlier forms" -> "have changed significantly from their earlier forms" ? The changes aren't really that drastic tbh.

Reply:
Fixed